src/main/java/com/google/sps/servlets/ItemFeedbackServlet.java

```diff
@@ -51,8 +51,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
 			boolean completed = Boolean.parseBoolean(request.getParameter("completed"));
 			int rating = Integer.parseInt(request.getParameter("rating"));
-
-			LearningItem learningItem = learningPathService.submitFeedback(pathId, ItemId, user.getId(), rating, completed);
+            // TODO add LearningSectionId
+			LearningItem learningItem = learningPathService.submitFeedback(pathId, 0, ItemId, user.getId(), rating, completed);
```

Comment:
Consider which component should be in charge of figuring this out.

 - The Servlet is mainly for handling user input (HTTP POST) data, and depends on various services for handling the steps of the request.
 - The `learningPathService` has direct access to the datastore

Instead of adding this is a parameter to be provided by the servlet, can the service instead e.g. load the item and figure out the sectionId this way?